src/components/CreateSecret/CreateSecret.scss

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The Tekton Authors
```

Comment:
Any existing files that are edited (including moved / renamed) in this PR should have `2019-2020`, any files that are newly introduced should have `2020`. I've also taken a note to see if we can automate this.